Stack8/github-zenhub-sdk
# Replace JitPack w/ On-Prem Maven Repository

### QA steps
- go to your `~/.gradle/gradle.properties` and add the following:
```
sonatypeUsername=gradle
sonatypePassword=<get password from 1pass>
```
- make a change to the interface in github-zenhub-sdk
  - I went to `ZenHubClient` and added a public dummy method
- run `./gradlew publish -Psnapshot` to update the snapshot
- go to the feature branch in ziro-cli
- in the gradle plugin tab, right click and hit "refresh dependencies" to pull in the latest snapshot
- verify the new method exists by going to a file like `ZenHubIssueService` and attempt to use the method
- revert the public dummy method and repeat the steps to ensure the method no longer appears
